Resources/Prototypes/Entities/Mobs/Species/slime.yml

```
@@ -54,7 +54,7 @@
         scaled: true
         damage:
           types:
-            Heat: 2
+            Heat: 4
```

```suggestion
            Heat: 2
```

This change is unnecessary, for similar reasons as above.

---

Alright, that's fair. This one was a bit of an afterthought for the PRs purpose anyway.